packages/aws-cdk/package.json

```
@@ -106,26 +106,26 @@
     "@aws-cdk/cloudformation-diff": "0.0.0",
     "@aws-cdk/cx-api": "0.0.0",
     "@aws-cdk/region-info": "0.0.0",
-    "@aws-sdk/client-appsync": "3.696.0",
-    "@aws-sdk/client-cloudformation": "3.696.0",
-    "@aws-sdk/client-cloudwatch-logs": "3.696.0",
-    "@aws-sdk/client-codebuild": "3.696.0",
-    "@aws-sdk/client-ec2": "3.696.0",
-    "@aws-sdk/client-ecr": "3.696.0",
-    "@aws-sdk/client-ecs": "3.696.0",
-    "@aws-sdk/client-elastic-load-balancing-v2": "3.696.0",
-    "@aws-sdk/client-iam": "3.696.0",
-    "@aws-sdk/client-kms": "3.696.0",
-    "@aws-sdk/client-lambda": "3.696.0",
-    "@aws-sdk/client-route-53": "3.696.0",
-    "@aws-sdk/client-s3": "3.696.0",
-    "@aws-sdk/client-secrets-manager": "3.696.0",
-    "@aws-sdk/client-sfn": "3.696.0",
-    "@aws-sdk/client-ssm": "3.696.0",
-    "@aws-sdk/client-sts": "3.696.0",
-    "@aws-sdk/credential-providers": "3.696.0",
-    "@aws-sdk/ec2-metadata-service": "3.696.0",
-    "@aws-sdk/lib-storage": "3.696.0",
+    "@aws-sdk/client-appsync": "^3.699.0",
```

Don't think it matters, but why the carets suddenly? I prefer to not include this change in this PR. 

```suggestion
    "@aws-sdk/client-appsync": "3.699.0",
```

---

Don't we need the carets to indicate to dependabot that it should keep it up to date? We used to have them for v2.